cli/lsp/lsp_extensions.rs

```diff
@@ -5,22 +5,10 @@
 ///!
 use deno_core::serde::Deserialize;
 use deno_core::serde::Serialize;
-use lsp_types::request::Request;
-use lsp_types::TextDocumentIdentifier;
+use lspower::lsp_types::TextDocumentIdentifier;
 
 #[derive(Debug, Deserialize, Serialize)]
```

Comment:
probably should be moved to "backend" (which I still don't like the name) and this file deleted.